content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -620,6 +620,11 @@ spec:
 
 PersistentVolumes binds are exclusive, and since PersistentVolumeClaims are namespaced objects, mounting claims with "Many" modes (`ROX`, `RWX`) is only possible within one namespace.
 
+### PersistentVolumes typed HostPath
+
+A hostPath PersistentVolume uses a file or directory on the Node to emulate network-attached storage.
+See [an example of HostPath typed volume](/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-a-persistentvolume).
```

Comment:
With this change now adding just two lines, I wonder if it makes sense to move it into the section headed `## {{% heading "whatsnext" %}}`. That's where I'd put it.

Second choice, move it after the list of volume types that ends on line 332.

Reply:
I would like to keep it up in the doc, instead of at the bottom. How about putting it under Persistent Volumes, line 364, before capacity? 332 seems wrong, because we don't want persistent volume to come after the example of HostPath, seems in reverse order. Tell me your thoughts. I am online and willing to update PR to get it approved.

Reply:
The whole page could use a rewrite to be honest. Those two places I mentioned feel like the least bad places to add this extra text.

Reply:
I think we can move it down to the bottom then. Would you be able to approve the changes then? Do you know who else will need to approve it before its merged. Asking as a first time contributor.

Reply:
I think this one is good to merge because we have too many types of persistent volumes on this page. Readers won't realize there are links to `hostPath` volume examples when reading this subsection.

Reply:
thanks @tengqm